ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -173,149 +164,6 @@ class TransactionCommitterSpec
     }
   }
 
-  "deduplicateCommand" should {
```

Comment:
These tests seem to have disappeared, missing file?

Reply:
Good catch, thank you. Added the file